### PR TITLE
adding ARK RTK GPS

### PR DESCRIPTION
**Pull request by samchamberlin**

### CODE REVIEW

File: en/uavcan/ark_rtk_gps.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# ARK RTK GPS
+
+ARK RTK GPS is an open source [UAVCAN](README.md) [RTK GPS](../gps_compass/rtk_gps.md), [u-blox F9P](https://www.u-blox.com/en/product/zed-f9p-module), magnetometer, barometer, IMU, buzzer, and safety switch module.
+
+![ARK RTK GPS](../../assets/hardware/gps/ark_rtk_gps.jpg)
```
Comment (hamishwillee) [inline en/uavcan/ark_rtk_gps.md:5]: This image is 1.35Mb, which is pretty large for the content. Can you shrink it a bit?
For example, this cuts off the dead space and reduces width to 1000px - only 140Kb ish.
ark_rtk_gps

**Comment by hamishwillee** [comment]
Other than the image size this looks good. 

More cross linking required:
- Add it to the table here: https://docs.px4.io/master/en/gps_compass/rtk_gps.html#supported-rtk-devices
- Add to table here: https://docs.px4.io/master/en/gps_compass/#supported-gnss-and-or-compass
- Add it to list of UAVCan cases mentioned here: https://docs.px4.io/master/en/gps_compass/#hardware-setup
- Add to list of UAVCAN compasses here: https://docs.px4.io/master/en/uavcan/#supported-hardware
- If you support the use case [RTK GPS Heading with Dual u-blox F9P]( https://docs.px4.io/master/en/gps_compass/u-blox_f9p_heading.html#rtk-gps-heading-with-dual-u-blox-f9p) please add your device to the list.

Probably I need to reduce all the required cross linking! But I think it helps users when they look from different places.

**Comment by hamishwillee** [comment]
@samchamberlin Ping. Did you see the comments above?

**Comment by samchamberlin** [comment]
Yeah sorry for no response! Been busy with other projects. I’ll take care
of this hopefully by this weekend. Thanks Hamish!

Sam

**Comment by hamishwillee** [comment]
Thanks @samchamberlin !

**Pull request merged**